packages/tinacms/src/hooks/formify/util.ts

```diff
@@ -362,7 +372,7 @@ export const getFieldAliasForBlueprint = (path: BlueprintPath[]) => {
     }
     return true
   })
-  return accum.reverse().slice(1).join('.')
+  return accum.reverse().join('.')
```

Comment:
Because of the `data` key, we had to "hop" over it, and we assumed it was always the second item `['getPostDocument', 'data', 'title']` for example. But now that it's gone: `['post', 'title']`, hence the slice being removed.